Src/Screens/ProfileScreen/Screens/MyPayment/Viewables/HeaderCard/PaymentHeaderNextCharge.swift

```diff
@@ -48,6 +51,7 @@ extension PaymentHeaderNextCharge: Viewable {
                 view.backgroundColor = .primaryBackground
             } else {
                 label.value = String(key: .PAYMENTS_CARD_NO_STARTDATE)
+                label.textColor = .darkGray
```

Comment:
Using textColor directly is prohibited, this comes directly from the textstyle instead.